src/main/java/com/ericsson/ei/erqueryservice/ERQueryService.java

```diff
@@ -105,7 +112,10 @@ private ResponseEntity sendRequestToER(String eventId, SearchOption searchOption
 
         String uri = request.getURI().toString();
         LOGGER.debug("The URL to ER is: {}", uri);
+    }
 
+    private ResponseEntity sendRequest()
+            throws ClientProtocolException, URISyntaxException, IOException {
```

Comment:
So I get the idea. What is the benefit of putting this  line in its own method?

Reply:
I don't know, ask Anders. Maybe I misunderstood him. You can see the feedback he gave above.

Reply:
@a-breid-ericsson Was your idea of puttin one statement inside a method? If it was, what is the benifit?

Reply:
I requested it to be 2 methods, now it became 3 :) before the prepare part was inside the try catch for executing the request, that was not needed. If the try catch part was in this method it would be more logical.

Reply:
I removed this method.